tchannel/tornado/peer.py

```diff
@@ -624,7 +624,7 @@ def send_with_retry(self, request, peer, attempt_times, retry_delay):
                 )
 
                 if not connection:
-                    raise
+                    raise protocol_error
```

Comment:
not a breaking change. this was previously a re-raise of the same exception, but since we're inside a coroutine, the yield could forget this exceptino